jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/SpringBootConfiguration.java

```diff
@@ -58,7 +58,7 @@ public static SpringBootConfiguration from(JavaProject project) {
       .managementPort(Optional.ofNullable(properties.getProperty("management.port")).map(Integer::parseInt).orElse(null))
       .serverPort(Integer.parseInt(properties.getProperty("server.port", DEFAULT_SERVER_PORT)))
       .serverKeystore(properties.getProperty("server.ssl.key-store"))
-      .managementHealthProbesEnabled(Boolean.parseBoolean(properties.getProperty("management.health.probes.enabled")))
+      .managementHealthProbesEnabled(Boolean.parseBoolean(properties.getProperty("management.endpoint.health.probes.enabled")))
```

Comment:
In order to keep backwards compatibility, do you think we should overwrite this field if version is less than 2.3.2 ? 

```java
// Parse minorVersion 
if (majorVersion < 3 && minorVersion < 4) {
  configBuilder.managementHealthProbesEnabled(Boolean.parseBoolean(properties.getProperty("management.health.probes.enabled")))
}
```

Reply:
sure, in addition to maintaining backward compatibility, wdyt to add a warning log notice, in case the user uses the old name to warn about the new name and a removal in the next minor release ?

Reply:
+1, sounds good to me

Reply:
ready for review